farm/data_handler/utils.py

```diff
@@ -36,6 +36,7 @@ def read_tsv(filename, quotechar='"', delimiter="\t", skiprows=None, columns=Non
         names=columns,
         skiprows=skiprows,
     )
+    df['label'] = df['label'].astype(str)
```

Comment:
Hey that is one way of doing it, that will solve the problem. I think it is slightly better to give a dtype=str argument to the pandas.read_csv directly, the automatic casting seems like wasted FLOPs : )
And we actually also want the input text fields to always be strings, so the dtype=str should be applied to all columns (and not just the label column).

Reply:
I just changed it as you requested. : )